activesupport/test/current_attributes_test.rb

```diff
@@ -0,0 +1,80 @@
+require "abstract_unit"
+require "active_support/current_attributes"
+
+Person = Struct.new(:name, :time_zone)
+
+class Current < ActiveSupport::CurrentAttributes
+  attribute :world, :account, :person
```

Comment:
I still think we could confuse people into thinking that the Active Model attributes type casting stuff is available here.

So since these are mostly `attributes` in name only — there's no dirty tracking, aliasing or other goodies from Active Model — perhaps we should just think of them as accessors?

```ruby
class Current < ActiveSupport::CurrentAccessors
  accesses :world, :account, :person # Or maybe even `attr_accessor`?
end
```